components/ui/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import { ButtonHTMLAttributes, PropsWithChildren } from "react";
+import styles from "./Button.module.css";
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  className?: string;
+}
+
+const Button = ({
+  type = "button",
+  className = "",
+  disabled = false,
+  children,
+  onClick = () => {},
```

Comment:
P2:
defaultValue로 빈 함수가 필요한 이유를 모르겠습니다~

```suggestion
  onClick,
```